packages/calcite-components/src/utils/interactive.tsx

```diff
@@ -125,7 +125,11 @@ function addInteractionListeners(element: HTMLElement): void {
 }
 
 function getParentElement(component: InteractiveComponent): ParentElement {
-  return interactiveElementToParent.get(component.el as InteractiveHTMLElement);
+  return (
+    interactiveElementToParent.get(
```

Comment:
Both this change and the other change are necessary for 'disabled' to consistently work in Firefox in my testing, but I don't fully understand why this part is necessary.